package.json

```diff
@@ -23,10 +23,10 @@
   "dependencies": {
     "@babel/polyfill": "^7.2.5",
     "@okta/jwt-verifier": "^1.0.0",
-    "@okta/okta-react": "^3.0.2",
+    "@okta/okta-react": "^1.4.0",
```

Comment:
Looks like we have an issue with webpack with upgraded version. Downgrading temporarily until the issue is resolved